app/gui/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -55,6 +57,8 @@ pub mod visualization_chooser;
 
 /// Default width and height of the visualization container.
 pub const DEFAULT_SIZE: (f32, f32) = (200.0, 200.0);
+/// Minimal allowed size of the visualization container.
+const MIN_SIZE: Vector2 = Vector2(200.0, 200.0);
```

Comment:
The `DEFAULT_SIZE` could also have Vector2 type.

Reply:
Done